test/PoseidonGoldenTests/GoldenTestsRunCommands.hs

```diff
@@ -876,11 +892,31 @@ testPipelineForge testDir checkFilePath = do
         , _forgeOutPacName   = Just "ForgePac17"
         , _forgePackageWise  = False
         , _forgeOutputPlinkPopMode = PlinkPopNameAsFamily
+        , _forgeOutputOrdered = False
     }
     let action17 = testLog (runForge forgeOpts17) >> patchLastModified testDir ("forge" </> "ForgePac17" </> "POSEIDON.yml")
     runAndChecksumFiles checkFilePath testDir action17 "forge" [
           "forge" </> "ForgePac17" </> "ForgePac17.janno"
         ]
+    let forgeOpts18 = ForgeOptions {
+          _forgeGenoSources  = [PacBaseDir $ testPacsDir </> "Schiffels_2016"]
+        , _forgeEntityInput  = [EntitiesDirect (fromRight [] $ readEntitiesFromString "POP3,<XXX004>,<XXX006>,<XXX003>")]
+        , _forgeSnpFile      = Nothing
+        , _forgeIntersect    = False
+        , _forgeOutFormat    = GenotypeFormatPlink
+        , _forgeOutMinimal   = False
+        , _forgeOutOnlyGeno  = False
+        , _forgeOutPacPath   = testDir </> "forge" </> "ForgePac18"
+        , _forgeOutPacName   = Just "ForgePac18"
+        , _forgePackageWise    = False
+        , _forgeOutputPlinkPopMode = PlinkPopNameAsFamily
+        , _forgeOutputOrdered = True
+    }
+    let action18 = testLog (runForge forgeOpts18) >> patchLastModified testDir ("forge" </> "ForgePac18" </> "POSEIDON.yml")
+    runAndChecksumFiles checkFilePath testDir action18 "forge" [
+          "forge" </> "ForgePac18" </> "ForgePac18.janno",
+          "forge" </> "ForgePac18" </> "ForgePac18.fam"
```

Comment:
Maybe the .bed file as well as its order also depends on on `_forgeOutputOrdered = True`.

Reply:
Oh good point. Yes, I will add that!